src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -300,7 +303,7 @@ def body(self):
 class NetworkLoadBalancerSubresourceScenarioTest(ResourceGroupVCRTestBase):
 
     def __init__(self, test_method):
-        super(NetworkLoadBalancerSubresourceScenarioTest, self).__init__(__file__, test_method)
+        super(NetworkLoadBalancerSubresourceScenarioTest, self).__init__(__file__, test_method, debug=True)
```

Comment:
these tend to sneak through easily, it almost might make sense to set them via environment variable :)

---
In reply to: [75965252](https://github.com/Azure/azure-cli/pull/735#discussion_r75965252) [](ancestors = 75965252)